test/Makefile

```diff
@@ -154,7 +154,11 @@ all_targets :=
 include ../Makefile.quiet
 
 
-ifdef CONFIG_HAVE_STATX
+ifdef CONFIG_HAVE_STATX 
+	test_srcs += statx.c
+endif
+
+ifdef CONFIG_HAVE_GLIBC_STATX 
```

Comment:
Please remove that trailing whitespace (a space char after "CONFIG_HAVE_GLIBC_STATX").